entity-framework/toc.yml

```diff
@@ -18,10 +18,29 @@
 
   - name: Entity Framework Core
     items:
-    - name: Overview
+    - name: Welcome!
       href: core/index.md
     - name: Take the EF Core 6 survey >>
       href: https://www.surveymonkey.com/r/XRL3JBL
+    - name: Getting started
+      items:
+      - name: EF Core Overview
+        items:
+        #- name: What is EF Core? <<<--- No Page
+        - name: Getting EF Core <<<--- formerly Install EF Core
```

Comment:
Can folks verify items with '<<<---" in this file which is not commented out with # at the start?
Some requires verification only. Some did not have a very specific instruction on what to do so requires guidance.

Reply:
Should have tagged @dotnet/efteam

Reply:
Install